scripts/prepare_kb.py

```diff
@@ -0,0 +1,26 @@
+import os
+from enum import Enum
+from copy_kb import copy_kb
+import sys
+
+
+class CopyKbPaths(Enum):
+    SCRIPTS = os.path.dirname(os.path.abspath(__file__))
+    OSTIS = os.path.split(SCRIPTS)[0]
+    KB = os.path.join(OSTIS, 'ims.ostis.kb')
+
+
+def main(copy_kb_path: str):
+    copy_kb_path = os.path.join(CopyKbPaths.OSTIS.value, os.path.split(copy_kb_path)[1])
+    CopyKbPaths.KB_COPY = os.path.join(CopyKbPaths.OSTIS.value, copy_kb_path)
```

Comment:
`CopyKbPaths.KB_COPY` will have `CopyKbPaths.OSTIS.value` twice in its path. It should be just set equal to `copy_kb_path` if we need to save this path in `CopyKbPaths.KB_COPY`.
```suggestion
    CopyKbPaths.KB_COPY = copy_kb_path
```